src/wscearth/views.py

```diff
@@ -47,8 +48,30 @@
 @app.route("/")
 @cache.cached(timeout=30)
 @flask_cachecontrol.cache_for(seconds=30)
-def positions():
-    """Render a positions map"""
+def index():
+    """Render a Google map"""
+    return flask.render_template("positions_map.html")
+
+
+@app.route("/api/path/<shortname>")
```

Comment:
We'll need/want to add some caching here.

Reply:
Good thought. Missed that one.